articles/connections/references/apple-native/guides/add-siwa-to-web-app.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Add Sign In with Apple to Your Web App
+description: Learn how to add native login functionality to your web app with Apple. 
+toc: true
+topics:
+  - authentication
+  - connections
+  - social
+  - apple
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - add-idp
+---
+# Add Sign In with Apple to Your Web App
+
+To set up and configure native Sign In with Apple, you will do the following:
+
+* [Complete the prerequisites](#prerequisites): Have an Apple Developer account (paid), an Auth0 tenant set up with a custom domain, and a web application configured to use Auth0 for authentication at that domain.  
+* [Register your app in your Apple Developer account](#register-your-app-in-your-apple-developer-account): Obtain Apple IDs and keys for the application connection settings in the Dashboard.
+* [Create your web app](#create-your-web-application): Including verifying your domain ownership with Apple and configuring Apple's email relay service, if necessary.
```

Comment:
This step feels more like a quickstart.  Is this article aimed at a configuration task, or providing a quickstart sample to developers?

Reply:
It is a how-to guide for configuring Sign In with Apple for a web app. The quickstarts are typically tutorials with sample code and we walk the developers through creating a working sample. We can remove the create app stuff here if that makes more sense.